msgid ""
msgstr ""
"Project-Id-Version: \n"
"Report-Msgid-Bugs-To: 'nvda-translations@groups.io'\n"
"POT-Creation-Date: 2024-05-24 10:24+0300\n"
"PO-Revision-Date: 2024-05-24 10:31+0300\n"
"Last-Translator: Heorhii Halas <nvda.translation.uk@gmail.com>\n"
"Language-Team: \n"
"Language: uk\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 3.4.4\n"
"X-Poedit-Basepath: AI-content-describer-main/addon\n"
"X-Poedit-SearchPath-0: globalPlugins\n"
"X-Poedit-SearchPath-1: installTasks.py\n"

#. Translators: The message displayed if during installation, there is an incompatable version already installed
#: addon\installTasks.py:20
msgid ""
"There appears to be an older version of this add-on installed that is "
"incompatable with future versions because the name has changed. It will be "
"removed the next time NVDA is restarted."
msgstr ""
"Схоже, що встановлено стару версію цього додатка, яка несумісна з наступними "
"версіями, оскільки назву було змінено. Її буде видалено під час наступного "
"перезапуску NVDA."

#. Translators: The title of the message dialog when the user is installing the add-on
#. Translators: The label for the category in NVDA settings
#. Translators: Summary for this add-on
#. Add-on summary, usually the user visible name of the addon.
#. Translators: Summary for this add-on
#. to be shown on installation and add-on information found in Add-ons Manager.
#: addon\installTasks.py:22
#: addon\globalPlugins\AIContentDescriber\__init__.py:94
#: addon\globalPlugins\AIContentDescriber\__init__.py:198
#: addon\globalPlugins\AIContentDescriber\dependency_checker.py:103
#: addon\globalPlugins\AIContentDescriber\dependency_checker.py:113
#: addon\globalPlugins\AIContentDescriber\dependency_checker.py:123
#: addon\globalPlugins\AIContentDescriber\dependency_checker.py:133
#: addon\globalPlugins\AIContentDescriber\dependency_checker.py:144
#: addon\globalPlugins\AIContentDescriber\dependency_checker.py:154
#: addon\globalPlugins\AIContentDescriber\dependency_checker.py:220
#: buildVars.py:23
msgid "AI Content Describer"
msgstr "Описувач вмісту за допомогою ШІ"

#. translators: the button in the settings dialog to open the model manager
#: addon\globalPlugins\AIContentDescriber\__init__.py:99
msgid "Manage &models"
msgstr "Керування &моделями"

#. translators: the label for the dropdown that lists the currently available models
#: addon\globalPlugins\AIContentDescriber\__init__.py:101
msgid ""
"Model (configure more in the manage models dialog, defaults to last used):"
msgstr ""
"Модель (налаштуйте більше у діалозі керування моделями, початково "
"використовується остання вибрана)"

#. Translators: The label for the option to open results in browseable dialogs
#: addon\globalPlugins\AIContentDescriber\__init__.py:103
msgid ""
"Open each result in a browseable dialog; Markdown will be rendered if "
"possible"
msgstr ""
"Відкривати кожен результат у діалозі, доступному для огляду; Markdown буде "
"показано, якщо це можливо"

#. Translators: The label for the checkbox to cash images and their descriptions in the settings dialog
#: addon\globalPlugins\AIContentDescriber\__init__.py:105
msgid "Remember/cache descriptions of each item to save API quota"
msgstr "Запам'ятати/кешувати описи кожного елемента, щоб заощадити квоту API"

#. Translators: The label for the checkbox that controls whether to optimize image uploads for size in the settings dialog
#: addon\globalPlugins\AIContentDescriber\__init__.py:107
msgid ""
"Optimize images for size, may speed up detection in some situations "
"(experimental)"
msgstr ""
"Оптимізувати зображення за розміром, може пришвидшити виявлення в деяких "
"ситуаціях (експериментально)"

#. translators: current focus
#: addon\globalPlugins\AIContentDescriber\__init__.py:152
msgid "Current focus"
msgstr "Поточний фокус"

#. translators: navigator object
#: addon\globalPlugins\AIContentDescriber\__init__.py:154
msgid "Navigator object"
msgstr "Об'єкт Навігатора"

#. translators: screenshot of entire window menu item
#: addon\globalPlugins\AIContentDescriber\__init__.py:156
msgid "Entire screen"
msgstr "Весь екран"

#. translators: picture from the local camera menu item
#: addon\globalPlugins\AIContentDescriber\__init__.py:158
msgid "Take a picture"
msgstr "Зробити знімок"

#: addon\globalPlugins\AIContentDescriber\__init__.py:161
msgid "Detect face position"
msgstr "Визначити положення обличчя"

#. self.detect_face_realtime_item = self.face_detection_menu.Append(wx.ID_ANY, _("Real-time face guidance"))
#: addon\globalPlugins\AIContentDescriber\__init__.py:163
msgid "Select camera"
msgstr "Виберіть камеру"

#: addon\globalPlugins\AIContentDescriber\__init__.py:164
msgid "Release the camera to make it usable by other applications"
msgstr "Зробити камеру доступною для використання іншими програмами"

#. translators: the label for the submenu that contains the options for face detection. Also informs the user that this feature does not require API access.
#: addon\globalPlugins\AIContentDescriber\__init__.py:166
msgid "Face Detection (no API required)"
msgstr "Виявлення обличчя (API не потрібен)"

#. translators: the name of the submenu used to select a model.
#: addon\globalPlugins\AIContentDescriber\__init__.py:177
msgid "Model"
msgstr "Модель"

#. Translators: message spoken when there is an attempt to recognize an object, but the screen curtain is running
#: addon\globalPlugins\AIContentDescriber\__init__.py:229
msgid ""
"Please disable windows screen curtain before using AI content describer."
msgstr "Будь ласка, вимкніть екранну завісу Windows перед використанням ШІ."

#: addon\globalPlugins\AIContentDescriber\__init__.py:243
msgid "Content is not visible"
msgstr "Вміст не видно"

#. Translators: Message spoken when the attempt to take a picture of an object fails
#: addon\globalPlugins\AIContentDescriber\__init__.py:254
msgid "Could not snap an image of the requested object"
msgstr "Не вдалося зробити знімок потрібного об’єкта"

#. py-3.11: On some machines, it is rare, but cv2 might not work as expected due to a variety of reasons none of which I can reproduce.
#. # In the meantime, establish a catchall exception and report it to the user so they can file an issue
#. Translators: The message spoken when there was an error with the face detection interface.
#: addon\globalPlugins\AIContentDescriber\__init__.py:267
#: addon\globalPlugins\AIContentDescriber\__init__.py:292
msgid ""
"Error initializing the face detection interface. Please consult the NVDA log "
"for more information. "
msgstr ""
"Помилка ініціалізації інтерфейсу виявлення облич. Будь ласка, зверніться до "
"журналу NVDA для отримання додаткової інформації. "

#. Translators: Message spoken when the face detection interface is loading
#: addon\globalPlugins\AIContentDescriber\__init__.py:271
#: addon\globalPlugins\AIContentDescriber\__init__.py:296
msgid ""
"Face detection interface is loading. Please try this command again after a "
"couple seconds."
msgstr ""
"Завантажується інтерфейс виявлення обличчя. Будь ласка, повторіть цю команду "
"через кілька секунд."

#. translators: message spoken when grabbing the content of the current window is not possible
#: addon\globalPlugins\AIContentDescriber\__init__.py:279
msgid "Could not get window content"
msgstr "Не вдалося отримати вміст вікна"

#. translators: message spoken when the picture could not be taken due to an unknown error
#: addon\globalPlugins\AIContentDescriber\__init__.py:303
#: addon\globalPlugins\AIContentDescriber\__init__.py:313
msgid ""
"The picture could not be taken. Please ensure that your camera is not in use "
"by another application and try again."
msgstr ""
"Не вдалося зробити знімок. Переконайтеся, що вашу камеру не використовує "
"інша програма й повторіть спробу."

#. Translators: the message spoken when the picture is taken but the file could not be written.
#: addon\globalPlugins\AIContentDescriber\__init__.py:308
msgid "The picture could not be saved."
msgstr "Зображення не вдалося зберегти."

#. Translators: Message spoken when the item copied to the clipboard is not an image
#: addon\globalPlugins\AIContentDescriber\__init__.py:321
#: addon\globalPlugins\AIContentDescriber\__init__.py:332
msgid "The item on the clipboard is not an image."
msgstr "Елемент у буфері обміну не є зображенням."

#. Translators: Message spoken when the image on the clipboard is not a format supported by the current description service
#: addon\globalPlugins\AIContentDescriber\__init__.py:326
#, python-brace-format
msgid ""
"Unsupported image format. Please copy another file to the clipboard that is "
"{formats}"
msgstr ""
"Непідтримуваний формат зображення. Будь ласка, скопіюйте в буфер обміну "
"інший файл {formats}"

#. Translators: Message spoken when the user attempts to describe something but they haven't yet provided an API key or base URL
#: addon\globalPlugins\AIContentDescriber\__init__.py:342
msgid ""
"To describe content, you must provide an API key or base URL in the AI image "
"describer category of the NVDA settings dialog. Please consult add-on help "
"for more information"
msgstr ""
"Щоб описати вміст, ви повинні надати ключ API або базову URL-адресу у "
"категорії \"Описувач вмісту за допомогою ШІ\" у діалозі налаштувань NVDA. "
"Будь ласка, зверніться до довідки додатка для отримання додаткової інформації"

#. Translators: Message spoken when a user attempts to describe something, but they haven't provided a prompt
#: addon\globalPlugins\AIContentDescriber\__init__.py:346
msgid ""
"To describe content, you must define a prompt by navigating to the AI image "
"describer category of the NVDA settings dialog. Please consult add-on help "
"for more information"
msgstr ""
"Щоб описати вміст, ви маєте визначити підказку, перейшовши до категорії "
"\"Описувач вмісту за допомогою ШІ\" у діалозі налаштувань NVDA. Будь ласка, "
"зверніться до довідки додатка для отримання додаткової інформації"

#. Translators: Message spoken after the beep - when we have started fetching the description
#: addon\globalPlugins\AIContentDescriber\__init__.py:350
#, python-brace-format
msgid "Retrieving description using {name}..."
msgstr "Отримання опису за допомогою {name}..."

#. Translators: Title of the browseable message
#: addon\globalPlugins\AIContentDescriber\__init__.py:354
msgid "Image description"
msgstr "Опис зображення"

#. translators: message spoken after the camera has been released successfully
#: addon\globalPlugins\AIContentDescriber\__init__.py:394
msgid "Success"
msgstr "Успішно"

#: addon\globalPlugins\AIContentDescriber\__init__.py:401
msgid ""
"Describe the image (or file path to an image) on the clipboard using AI."
msgstr ""
"Описати зображення (або шлях до файлу зображення) у буфері обміну за "
"допомогою ШІ."

#: addon\globalPlugins\AIContentDescriber\__init__.py:405
msgid "Describe the contents of the current navigator object using AI."
msgstr "Описати вміст поточного об’єкта навігатора за допомогою ШІ."

#: addon\globalPlugins\AIContentDescriber\__init__.py:409
msgid "Describe the contents of the currently focused item using AI."
msgstr "Описати вміст поточного об’єкта навігатора за допомогою ШІ."

#: addon\globalPlugins\AIContentDescriber\__init__.py:413
msgid "Take a screenshot, then describe it using AI."
msgstr "Зробити знімок екрана, а потім описати його за допомогою ШІ."

#: addon\globalPlugins\AIContentDescriber\__init__.py:417
msgid ""
"Pop up a menu asking whether to describe the current focus, navigator "
"object, or entire screen with AI."
msgstr ""
"Відкриває спливаюче меню із запитом, чи потрібно описати поточний фокус, "
"об’єкт навігатора чи весь екран за допомогою ШІ."

#: addon\globalPlugins\AIContentDescriber\__init__.py:421
msgid "Snap a picture using the selected camera, then describe it using AI."
msgstr ""
"Зробити знімок за допомогою вибраної камери, а потім описати його за "
"допомогою ШІ."

#: addon\globalPlugins\AIContentDescriber\__init__.py:425
msgid ""
"Describe the position of the face in the frame using the selected camera, if "
"applicable."
msgstr ""
"Описати положення обличчя в кадрі за допомогою вибраної камери, якщо це "
"можливо."

#. Translators: the message shown when required dependencies were not found
#: addon\globalPlugins\AIContentDescriber\dependency_checker.py:105
msgid ""
"Some of the dependencies required for this NVDA add-on to run are not "
"available on your computer. Would you like to download them now? NVDA will "
"need to be restarted when this process completes."
msgstr ""
"Деякі залежності, необхідні для запуску цього додатка NVDA, відсутні на "
"вашому комп’ютері. Бажаєте завантажити їх зараз? Після завершення цього "
"процесу потрібно буде перезапустити NVDA."

#. Translators: the message shown when the wrong dependency versions were found
#: addon\globalPlugins\AIContentDescriber\dependency_checker.py:115
msgid ""
"Some of the dependencies required for this NVDA add-on to run are outdated. "
"Would you like to try to install them now? NVDA will need to be restarted "
"when complete."
msgstr ""
"Деякі залежності, необхідні для запуску цього додатка NVDA, застарілі. "
"Бажаєте спробувати встановити їх зараз? Після завершення встановлення "
"потрібно буде перезапустити NVDA."

#. Translators: the message shown when the addon has successfully retrieved the new dependencies, asking the user whether they would like to delete the old ones
#: addon\globalPlugins\AIContentDescriber\dependency_checker.py:125
msgid ""
"Would you like to delete the old dependencies from the configuration? This "
"question will be asked once. If you are running multiple versions of NVDA "
"interchangeably, select no."
msgstr ""
"Ви бажаєте видалити застарілі залежності з конфігурації? Це запитання буде "
"задане один раз. Якщо ви використовуєте кілька версій NVDA одночасно, "
"виберіть \"ні\"."

#. Translators: Message shown when NVDA is about to restart after the dependencies have been downloaded.
#: addon\globalPlugins\AIContentDescriber\dependency_checker.py:135
msgid ""
"The dependencies have been downloaded successfully. NVDA will now restart "
"for the changes to take affect"
msgstr ""
"Залежності успішно завантажено. NVDA буде перезавантажено, щоб зміни набули "
"чинності"

#. Translators: Message displayed when there isn't yet libraries to support the running version of NVDA.
#: addon\globalPlugins\AIContentDescriber\dependency_checker.py:146
msgid ""
"Unfortunately, there doesn't yet seem to be dependencies available for the "
"running version of NVDA. Please submit an issue in our bug tracker or "
"contact the developers. In the meantime, you may wish to disable the add-on "
"to  surpress this  message on startup."
msgstr ""
"На жаль, для поточної версії NVDA ще немає доступних залежностей. Будь "
"ласка, повідомте про проблему у нашому баг-трекері або зв'яжіться з "
"розробниками. Тим часом, ви можете вимкнути додаток, щоб прибрати це "
"повідомлення під час запуску."

#. Translators: Message shown when dependencies fail to download.
#: addon\globalPlugins\AIContentDescriber\dependency_checker.py:156
msgid ""
"Unfortunately, there was a problem downloading the dependencies required to "
"run this add-on. Please consult the NVDA log for more details. "
msgstr ""
"На жаль, виникла проблема з завантаженням залежностей, необхідних для "
"запуску цього додатка. Щоб дізнатися більше, зверніться до журналу NVDA. "

#. translators: error
#: addon\globalPlugins\AIContentDescriber\description_service.py:36
#: addon\globalPlugins\AIContentDescriber\description_service.py:74
msgid "error"
msgstr "помилка"

#. translators: message spoken when we can't connect (error with connection)
#: addon\globalPlugins\AIContentDescriber\description_service.py:42
#: addon\globalPlugins\AIContentDescriber\description_service.py:87
msgid "error making connection"
msgstr "помилка з’єднання"

#. translators: message spoken when the connection is refused by our target
#: addon\globalPlugins\AIContentDescriber\description_service.py:49
#: addon\globalPlugins\AIContentDescriber\description_service.py:94
msgid "error, connection refused by target"
msgstr "помилка, кінцевий адресат відхилив підключення"

#. translators: the description for the GPT4 vision model in the model configuration dialog
#: addon\globalPlugins\AIContentDescriber\description_service.py:245
msgid ""
"The GPT4 model from OpenAI, previewed with vision capabilities. As of April "
"2024,  this model has been superseded by GPT4 turbo which has consistently "
"achieved better metrics in tasks involving visual understanding."
msgstr ""
"Модель GPT4 від OpenAI, представлена з можливостями сприйняття образів. "
"Станом на квітень 2024 року, цю модель замінила модель GPT4 Turbo, яка "
"показала кращі результати у завданнях, що вимагають розуміння візуальної "
"інформації."

#. translators: the description for the GPT4 turbo model in the model configuration dialog
#: addon\globalPlugins\AIContentDescriber\description_service.py:253
msgid ""
"The next generation of the original GPT4 vision preview, with enhanced "
"quality and understanding."
msgstr ""
"Наступне покоління оригінального GPT4 Vision з можливостями сприймання, з "
"покращеною якістю та розумінням."

#. translators: the description for the GPT4 omni model in the model configuration dialog
#: addon\globalPlugins\AIContentDescriber\description_service.py:261
msgid ""
"OpenAI's first fully multimodal model, released in May 2024. This model has "
"the same high intelligence as GPT4 and GPT4 turbo, but is much more "
"efficient, able to generate text at twice the speed and at half the cost."
msgstr ""
"Перша повністю мультимодальна модель OpenAI, випущена в травні 2024 року. Ця "
"модель має такий самий високий інтелект, як GPT4 і GPT4 turbo, але набагато "
"ефективніша, здатна генерувати текст удвічі швидше і вдвічі дешевше."

#. translators: the description for the Google Gemini pro vision model in the model configuration dialog
#: addon\globalPlugins\AIContentDescriber\description_service.py:274
msgid "Google's Gemini model with vision capabilities."
msgstr "Модель Gemini від Google з можливостями сприйняття образів."

#. translators: message spoken when Google gemini encounters an error with the format or content of the input.
#: addon\globalPlugins\AIContentDescriber\description_service.py:311
#, python-brace-format
msgid "Gemini encountered an error: {code}, {msg}"
msgstr "Gemini зіткнувся з помилкою: {code}, {msg}"

#. translators: message spoken when Claude encounters an error with the format or content of the input.
#: addon\globalPlugins\AIContentDescriber\description_service.py:376
#, python-brace-format
msgid "Claude encountered an error. {err}"
msgstr "Claude зіткнувся з помилкою. {err}"

#: addon\globalPlugins\AIContentDescriber\description_service.py:383
msgid "Anthropic's most powerful model for highly complex tasks."
msgstr "Потужна модель Anthropic для надскладних завдань."

#: addon\globalPlugins\AIContentDescriber\description_service.py:389
msgid ""
"Anthropic's model with Ideal balance of intelligence and speed, excels for "
"enterprise workloads."
msgstr ""
"Модель Anthropic з ідеальним балансом інтелекту та швидкості ідеально "
"підходить для корпоративних робочих навантажень."

#: addon\globalPlugins\AIContentDescriber\description_service.py:395
msgid ""
"Anthropic's fastest and most compact model for near-instant responsiveness"
msgstr ""
"Найшвидша та найкомпактніша модель Anthropic для майже миттєвої реакції"

#. translators: the description for the llama.cpp option in the model configuration dialog
#: addon\globalPlugins\AIContentDescriber\description_service.py:409
msgid ""
"llama.cpp is a state-of-the-art, open-source solution for running large "
"language models locally and in the cloud.\n"
"This add-on integration assumes that you have obtained llama.cpp from Github "
"and an image capable model from Huggingface or another repository, and that "
"a server is currently running to handle requests. Though the process for "
"getting this working is largely a task for the user that knows what they are "
"doing, you can find basic steps in the add-on documentation."
msgstr ""
"llama.cpp - це сучасне рішення з відкритим вихідним кодом для запуску "
"великих мовних моделей локально та у хмарі.\n"
"Інтеграція цього додатка передбачає, що ви отримали llama.cpp з Github і "
"модель з підтримкою зображень з Huggingface або іншого репозиторію, а також, "
"що наразі працює сервер для обробки запитів. Хоча процес інтеграції "
"здебільшого є завданням для користувача, який знає, що він робить, ви можете "
"знайти основні кроки в документації до додатка."

#: addon\globalPlugins\AIContentDescriber\description_service.py:439
#, python-brace-format
msgid ""
"Image recognition response appears to be malformed.\n"
"{response}"
msgstr ""
"Здається, відповідь розпізнавання зображень має неправильний формат.\n"
"{response}"

#. translators: the title for the dialog that selects the camera
#: addon\globalPlugins\AIContentDescriber\face_view.py:24
msgid "Choose a camera"
msgstr "Вибрати камеру"

#. translators: the label for the available cameras combo box
#: addon\globalPlugins\AIContentDescriber\face_view.py:34
msgid "&Available cameras"
msgstr "&Доступні камери"

#. translators: the message spoken to prompt the user to trigger the command again, after the camera has been chosen
#. # annoying behavior, this is a limitation of the way we're doing the UI. Yay for rapid prototyping!
#. release the old capture object (if one exists)
#: addon\globalPlugins\AIContentDescriber\face_view.py:42
msgid "Camera selected. You may now trigger the command again."
msgstr "Камеру вибрано. Тепер ви можете знову запустити команду."

#. Translators: the following messages correspond to the instructions provided to the user for facial detection. They are chosen and then concatinated together based on certain conditions, so please replicate the whitespace.
#: addon\globalPlugins\AIContentDescriber\face_view.py:79
msgid "far to the left of "
msgstr "набагато лівіше від "

#: addon\globalPlugins\AIContentDescriber\face_view.py:81
msgid "to the left of "
msgstr "ліворуч від "

#: addon\globalPlugins\AIContentDescriber\face_view.py:83
msgid "slightly to the left of "
msgstr "трохи ліворуч від "

#: addon\globalPlugins\AIContentDescriber\face_view.py:85
msgid "slightly to the right of "
msgstr "трохи праворуч від "

#: addon\globalPlugins\AIContentDescriber\face_view.py:87
msgid "to the right of "
msgstr "праворуч від "

#: addon\globalPlugins\AIContentDescriber\face_view.py:89
msgid "far to the right of "
msgstr "набагато правіше від "

#: addon\globalPlugins\AIContentDescriber\face_view.py:91
msgid "far below"
msgstr "занадто низько"

#: addon\globalPlugins\AIContentDescriber\face_view.py:93
msgid "below"
msgstr "нижче"

#: addon\globalPlugins\AIContentDescriber\face_view.py:95
msgid "slightly below"
msgstr "трохи нижче"

#: addon\globalPlugins\AIContentDescriber\face_view.py:97
msgid "slightly above"
msgstr "трохи вище"

#: addon\globalPlugins\AIContentDescriber\face_view.py:99
msgid "above"
msgstr "вище"

#: addon\globalPlugins\AIContentDescriber\face_view.py:101
msgid "far above"
msgstr "набагато вище"

#. Translators: Message spoken when the user's face is clearly in view
#: addon\globalPlugins\AIContentDescriber\face_view.py:103
msgid "Face clearly in view"
msgstr "Обличчя чітко видно"

#: addon\globalPlugins\AIContentDescriber\face_view.py:104
msgid " and "
msgstr " і "

#: addon\globalPlugins\AIContentDescriber\face_view.py:104
msgid "the center"
msgstr "в центрі"

#. translators: the message spoken when footage could not be captured from the camera during facial detection
#: addon\globalPlugins\AIContentDescriber\face_view.py:110
msgid ""
"Failed to interface with the camera. Please ensure it is not in use by "
"another application, then try again."
msgstr ""
"Не вдалося встановити зв'язок з інтерфейсом камери. Переконайтеся, що його "
"не використовує інша програма, а потім спробуйте ще раз."

#. translators: message spoken when the face detection failed because we are on the desktop - a rarely encountered windows bug
#: addon\globalPlugins\AIContentDescriber\face_view.py:124
msgid ""
"The footage from the camera is too blurry. Try switching your focus away "
"from the desktop, then try this command again."
msgstr ""
"Зображення з камери занадто розмите. Спробуйте переключити фокус з робочого "
"столу, а потім спробуйте виконати цю команду ще раз."

#. translators: message spoken when the face detection fails because the camera encountered a blurry image
#: addon\globalPlugins\AIContentDescriber\face_view.py:127
msgid ""
"The footage from the camera is too blurry. Please ensure that it is not "
"covered up and that your surroundings have proper lighting. {}"
msgstr ""
"Зображення з камери занадто розмите. Будь ласка, переконайтеся, що камера не "
"закрита, а навколишнє середовище має належне освітлення. {}"

#. translators: the message spoken when there was no face found in the frame
#: addon\globalPlugins\AIContentDescriber\face_view.py:132
msgid ""
"No face detected. Please ensure your face is in the frame and that your "
"camera is not covered up."
msgstr ""
"Обличчя не виявлено. Будь ласка, переконайтеся, що ваше обличчя потрапило в "
"кадр і що ваша камера не закрита."

#. translators: the message spoken when more than one face was detected
#: addon\globalPlugins\AIContentDescriber\face_view.py:135
msgid ""
"{len(faces)} faces detected near the frame. Please try for another angle "
"with fewer background objects."
msgstr ""
"Виявлено {len(faces)} облич. Будь ласка, спробуйте інший ракурс із меншою "
"кількістю предметів на задньому плані."

#. translators: the message that gets spoken when facial detection cannot find any cameras
#: addon\globalPlugins\AIContentDescriber\face_view.py:153
#: addon\globalPlugins\AIContentDescriber\face_view.py:174
msgid "No camera found on your system. Please connect one and try again."
msgstr ""
"У вашій системі не знайдено жодної камери. Будь ласка, підключіть хоча б "
"одну камеру та спробуйте ще раз."

#. translators: message spoken when the camera that was being used before has disconnected
#: addon\globalPlugins\AIContentDescriber\face_view.py:164
msgid " is no longer available. Switching to the other on the system."
msgstr " більше не доступний. Перемикаюсь на інший у системі."

#. translators: message spoken when the camera becomes unavailable between the user selecting it and triggering it again
#: addon\globalPlugins\AIContentDescriber\face_view.py:185
msgid "Camera unavailable. Please try selecting another"
msgstr "Камера недоступна. Будь ласка, спробуйте вибрати іншу"

#: addon\globalPlugins\AIContentDescriber\face_view.py:188
msgid ""
"Starting the face detection interface. This may take a few seconds. Please "
"wait."
msgstr ""
"Запуск інтерфейсу виявлення обличчя. Це може зайняти кілька секунд. Будь "
"ласка, зачекайте."

#. translators: message spoken when we could not interface with the chosen camera
#: addon\globalPlugins\AIContentDescriber\face_view.py:192
msgid "Failed to interface with the chosen camera"
msgstr "Не вдалося з’єднатися з інтерфейсом вибраної камери"

#. translators: the title of the model configuration dialog.
#: addon\globalPlugins\AIContentDescriber\model_configuration.py:16
msgid "Model configuration"
msgstr "Конфігурація моделі"

#. translators: the button in the model configuration dialog that provides information on the selected model
#: addon\globalPlugins\AIContentDescriber\model_configuration.py:29
msgid "About this model"
msgstr "Про цю модель"

#. Translators: The label for the API key field in the model configuration dialog
#: addon\globalPlugins\AIContentDescriber\model_configuration.py:33
msgid "API key"
msgstr "Ключ API"

#. Translators: The label for the base URL field in the model configuration dialog
#: addon\globalPlugins\AIContentDescriber\model_configuration.py:37
msgid "Base URL"
msgstr "Базовий URL"

#. Translators: The label for the prompt field in the model configuration dialog
#: addon\globalPlugins\AIContentDescriber\model_configuration.py:41
msgid "Prompt"
msgstr "Підказка"

#. Translators: The label for the button that resets the prompt to its default in the settings dialog
#: addon\globalPlugins\AIContentDescriber\model_configuration.py:43
msgid "Reset prompt to default"
msgstr "Скинути підказку до початкових"

#. Translators: The label for the maximum tokens chooser in the model configuration dialog
#: addon\globalPlugins\AIContentDescriber\model_configuration.py:47
msgid "Maximum tokens"
msgstr "Максимум токенів"

#. Translators: The label for the timeout chooser in the model configuration dialog
#: addon\globalPlugins\AIContentDescriber\model_configuration.py:51
msgid "Seconds to wait for a response before timing out"
msgstr "Секунди очікування відповіді перед закінченням часу очікування"

#. translators: the title for the about model message box
#: addon\globalPlugins\AIContentDescriber\model_configuration.py:93
msgid "About model"
msgstr "Про модель"

#. Add-on description
#. Translators: Long description to be shown for this add-on on add-on information from add-ons manager
#: buildVars.py:26
msgid ""
"This add-on makes it possible to describe the focus object, navigator "
"object, or screen using popular vision capable AI language models, like "
"Claude, Gemini, or GPT4.\n"
"It also lets one understand where their face is positioned in the frame of a "
"connected camera.\n"
"Though content descriptions are quite detailed, they may not always be "
"completely accurate or reflect real world information.\n"
"To begin with GPT, head to https://platform.openai.com/account/api-keys and "
"create an account, then create a key for interacting with the API. See add-"
"on documentation for more information on this.\n"
"Then, choose the \"AI content describer\" category from NVDA's settings "
"dialog -> manage models and enter your API key.\n"
"Press NVDA+shift+i to pop up a menu asking how you wish to describe based on "
"the current position, or NVDA+shift+u to describe the navigator object, or "
"NVDA+shift+y for an image that has been copied to the clipboard such as in "
"windows explorer. Other keystrokes are customizable from the input gestures "
"dialog."
msgstr ""
"Цей додаток дає змогу описувати об’єкт у фокусі, об’єкт навігатора або екран "
"за допомогою популярних мовних моделей штучного інтелекту з підтримкою зору, "
"таких як Claude, Gemini або GPT4.\n"
"Це також дозволяє зрозуміти, де розташоване обличчя в кадрі за допомогою "
"підключеної камери.\n"
"Хоча описи вмісту досить детальні, вони не завжди можуть бути повністю "
"точними або відображати реальну інформацію.\n"
"Для початку роботи з GPT перейдіть на сторінку https://platform.openai.com/"
"account/api-keys і створіть обліковий запис, а потім створіть ключ для "
"взаємодії з API. Додаткову інформацію про це див. у документації додатка.\n"
"Потім у діалозі налаштувань NVDA виберіть категорію \"Описувач вмісту за "
"допомогою ШІ\" і введіть свій ключ API.\n"
"Натисніть NVDA+shift+i, щоб викликати меню із запитом про спосіб опису на "
"основі поточної позиції, або NVDA+shift+u, щоб описати об'єкт навігатора, "
"або NVDA+shift+y для зображення, скопійованого до буфера обміну, наприклад, "
"у провіднику Windows. Інші комбінації клавіш можна налаштувати у діалозі "
"жести вводу."
